src/test/java/io/quarkus/registry/app/admin/AdminApiTest.java

```diff
@@ -179,8 +179,8 @@ void validate_input() {
                 .then()
                 .statusCode(HttpURLConnection.HTTP_BAD_REQUEST)
                 .contentType(ContentType.JSON)
-                .body("parameter-violations.message", hasItem("X-Platform header missing"),
-                        "parameter-violations.message", hasItem("Body payload is missing"));
+                .body("violations.message", hasItem("X-Platform header missing"),
+                        "violations.message", hasItem("Body payload is missing"));
```

Comment:
@gsmet while migrating to RestEasy Reactive I noticed this small change in the output of an Hibernate Validation JAX-RS method. Is this expected?

/cc @geoand

Reply:
Yeah, I wasn't too happy about it either. I mentioned it a long time ago.